internal/pkg/cli/cli.go

```diff
@@ -181,3 +184,40 @@ type storeReader interface {
 	archer.ApplicationLister
 	archer.ApplicationGetter
 }
+
+// returns true if error type is stack set not exist.
+func stackSetNotExistErr(err error) bool {
+	for {
+		if err == nil {
+			return false
+		}
+		aerr, ok := err.(awserr.Error)
+		if !ok {
+			return stackSetNotExistErr(errors.Unwrap(err))
+		}
+		if aerr.Code() != "StackSetNotFoundException" {
+			return stackSetNotExistErr(errors.Unwrap(err))
+		}
+		return true
+	}
+}
+
+// returns true if error type is stack not exist.
+func stackNotExistErr(err error) bool {
+	for {
+		if err == nil {
+			return false
+		}
+		aerr, ok := err.(awserr.Error)
+		if !ok {
+			return stackNotExistErr(errors.Unwrap(err))
```

Comment:
why do we need recursive calls here? Seems like `Unwrap` itself is already recursive: https://golang.org/src/errors/wrap.go?s=372:400#L4

Same for the `stackSetNotExistErr` above

Reply:
Oh...actually it is not. https://play.golang.org/p/KX468fAuxms